manifests/install.pp

```diff
@@ -1,20 +1,18 @@
 # See README.md for usage information
 class sssd::install (
+  $ensure                = $sssd::ensure,
```

Comment:
I don't see why this change is necessary.  The old variable name seems fine.
I would't want to break anyone's configuration, that is currently using this class directly...